.gitignore

```diff
@@ -1,5 +1,7 @@
 /.npmrc
 
+.domains
```

Comment:
This file should be visible

Reply:
Why? I put custom domain to it as developer